### PR TITLE
Add reworked version of Markov cog

### PR DESCRIPTION
**Pull request by tigattack**
# Initial Checklist
- [x] Has @tigattack / @issy been added as a reviewer?
- [x] If applicable, have the relevant project(s), milestone(s), and label(s) been applied?
- [x] If applicable, have you added details of the cog to the readme as per [README.md](https://github.com/rHomelab/LabBot-Cogs/blob/main/README.md#cog-summaries)?

# Details
**Does this resolve an issue?**
No.

### Features / Fixes
* Adds Markov cog based on [CBD-Cogs/Markov](https://gitlab.com/CrunchBangDev/cbd-cogs/-/tree/master/Markov).

### Breaking Changes
* N/A

## Additional
N/A

### CODE REVIEW

**Comment by tigattack** [comment]
Quite a large diff, not sure if anyone's going to want to review this.

The original code was functional and my changes have been through basic testing. My changes can be seen in 729a14e...e282c6c.

If nobody reviews after a week I'll just send it and see what happens 👀

File: README.md [hunk 1]
```diff
@@ -11,39 +11,43 @@ Cogs for the [RED](https://github.com/Cog-Creators/Red-DiscordBot/)-based [Homel
...
-    - [Roleinfo](#roleinfo)
-    - [Sentry](#sentry)
-    - [Timeout](#timeout)
-    - [Verify](#verify)
-    - [xkcd](#xkcd)
-  - [License](#license)
-  - [Contributing](#contributing)
-    - [Linting your code](#linting-your-code)
-    - [Making changes](#making-changes)
+* [LabBot Cogs](#labbot-cogs)
```
Comment (issy) [inline README.md:14]: IMO more consistent to keep these as dashes, like the rest of the file

File: markov/__init__.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from redbot.core.utils import get_end_user_data_statement
+
+from .markov import Markov
+
+__red_end_user_data_statement__ = get_end_user_data_statement(__file__)
+
+async def setup(bot):
+    bot.add_cog(Markov(bot))
```
Comment (issy) [inline markov/__init__.py:8]: This will need to be awaited for the `3.5.1` upgrade

File: markov/markov.py [hunk 3]
```diff
@@ -0,0 +1,476 @@
...
+
+import discord
+from redbot.core import Config, checks, commands
+from redbot.core.utils.mod import is_mod_or_superior
+
+log = logging.getLogger("red.rhomelab.markov")
+
+__all__ = ["UNIQUE_ID", "Markov"]
+
+UNIQUE_ID = 0x6D61726B6F76
```
Comment (issy) [inline markov/markov.py:14]: Probably want to change this ID as it's a fork of an existing cog with (presumably) the same name

**Comment by tigattack** [comment]
Thanks Issy :pray:

**Review by issy — APPROVED** [review]

**Pull request merged**